### PR TITLE
New blog

### PR DESCRIPTION
**Pull request by webuimaker**
can you write a new post, as an example? it will help teach me how to do it  -- DONE [I have added test post in markdown format]
could you give all files 2-spaces for indentation?[done please confirm]

### CODE REVIEW

File: _posts/2019-04-15-test.md [hunk 1]
```diff
@@ -8,18 +8,34 @@ excerpt_separator: <!--more-->
 dsq_thread_id:
   - "2735760452"
```
Comment (arshaw) [inline _posts/2019-04-15-test.md]: how did you decide what this `dsq_thread_id ` was?

**Comment by webuimaker** [comment]
Hi Adam,

This is  unique id for ref of post.

**Comment by arshaw** [comment]
@webuimaker, i know it's the unique ID, but how did you *create* the unique ID? Did you get it from the disqus website? did you press random numbers on your keyboard? I need to know how you CREATED it!

**Comment by webuimaker** [comment]
Hi Adam,
This id was already in old post, so I have added this random myself.
Thanks
Vivek

**Pull request merged**